com.unity.render-pipelines.universal/Shaders/PostProcessing/LensFlareDataDriven.shader

```diff
@@ -0,0 +1,149 @@
+            ZTest Always
+
+            HLSLPROGRAM
+
+            #pragma target 2.0
+            #pragma vertex vert
+            #pragma fragment frag
+
+            #pragma exclude_renderers gles
+
+            #pragma multi_compile_fragment _ FLARE_CIRCLE FLARE_POLYGON
+            #pragma multi_compile_fragment _ FLARE_INVERSE_SDF
+            #pragma multi_compile_vertex _ FLARE_OCCLUSION
+
+            #include "Packages/com.unity.render-pipelines.core/ShaderLibrary/Common.hlsl"
+
+            #include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/Core.hlsl"
+            #include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/DeclareDepthTexture.hlsl"
+            #include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/UnityInput.hlsl"
+            #include "Packages/com.unity.render-pipelines.universal/Shaders/PostProcessing/Common.hlsl"
+
+            #include "Packages/com.unity.render-pipelines.core/Runtime/PostProcessing/Shaders/LensFlareHDRPCommon.hlsl"
+
+            ENDHLSL
+        }
+        // Premultiply
+        Pass
+        {
+            Name "LensFlarePremultiply"
+            Tags { "Queue" = "Transparent" "RenderType" = "Transparent" "RenderPipeline" = "UniversalPipeline" }
+            LOD 100
+
+            Blend One OneMinusSrcAlpha
+            ColorMask RGB
+            ZWrite Off
+            Cull Off
+            ZTest Always
+
+            HLSLPROGRAM
+
+            #pragma target 2.0
+            #pragma vertex vert
+            #pragma fragment frag
+
+            #pragma exclude_renderers gles
+
+            #pragma multi_compile_fragment _ FLARE_CIRCLE FLARE_POLYGON
+            #pragma multi_compile_fragment _ FLARE_INVERSE_SDF
+            #pragma multi_compile_vertex _ FLARE_OCCLUSION
+
+            #include "Packages/com.unity.render-pipelines.core/ShaderLibrary/Common.hlsl"
+
+            #include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/Core.hlsl"
+            #include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/DeclareDepthTexture.hlsl"
+            #include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/UnityInput.hlsl"
+            #include "Packages/com.unity.render-pipelines.universal/Shaders/PostProcessing/Common.hlsl"
+
+            #include "Packages/com.unity.render-pipelines.core/Runtime/PostProcessing/Shaders/LensFlareHDRPCommon.hlsl"
+
+            ENDHLSL
+        }
+        // Lerp
+        Pass
+        {
+            Name "LensFlareLerp"
+            Tags { "Queue" = "Transparent" "RenderType" = "Transparent" "RenderPipeline" = "UniversalPipeline" }
+            LOD 100
+
+            Blend SrcAlpha OneMinusSrcAlpha
+            ColorMask RGB
+            ZWrite Off
+            Cull Off
+            ZTest Always
+
+            HLSLPROGRAM
+
+            #pragma target 2.0
+            #pragma vertex vert
+            #pragma fragment frag
+
+            #pragma exclude_renderers gles
```

Comment:
We need support for GLES2. Let's remove this line and try to in the editor and see if there are any compilation errors.